packages/shared/src/logger.ts

```diff
@@ -10,7 +10,8 @@ export const isDebug = () => {
   if (!process.env.DEBUG) {
     return false;
   }
-
+  
+  logger.level = 'verbose'; // support `process.env.DEBUG` in e2e
```

Comment:
?

Reply:
is this intended

Reply:
yeah, we cannot set logger level in e2e cases  by `process.env.DEBUG`

Reply:
okay I get it, can we move it to a separate PR

Reply:
one PR one change

Reply:
okay

Reply:
https://github.com/web-infra-dev/rsbuild/pull/2117